### PR TITLE
KIALI-3182 Migrate Tabs to PF4

### PR DESCRIPTION
**Pull request by xeviknal**
** Describe the change **
It migrates Tabs from Workload/App/Service Detail pages to PF4 components.

** Issue reference **
https://issues.jboss.org/browse/KIALI-2631

** Backwards compatible? **
yes

** TODO **

- [x] Traces tab with external link pending to implement (https://github.com/patternfly/patternfly-react/issues/2623#issuecomment-517211482)
- [x] Add bounderyException to tabs in the ServiceDetailsInfo Component (https://github.com/kiali/kiali-ui/pull/1349/files#diff-7cdcbd07abd513dbb734e4cfdb5aa81aL210)
- [ ] Change the background of each tab content to match expected mockup from UI (https://marvelapp.com/i9h9g7j/screen/59315434). Perhaps not part of this PR?

** Screenshot **
Screenshot of Kiali Console (6)

Screenshot of Kiali Console (7)

Screenshot of Kiali Console (8)

### CODE REVIEW

File: src/components/BreadcrumbView/BreadcrumbView.tsx [hunk 1]
```diff
@@ -97,37 +97,39 @@ export class BreadcrumbView extends React.Component<BreadCumbViewProps, BreadCum
...
-          <Link to={`/${pathItem}?namespaces=${namespace}`} onClick={this.cleanFilters}>
-            Namespace: {namespace}
-          </Link>
-        </Breadcrumb.Item>
-        {isIstio && (
-          <Breadcrumb.Item componentClass="span">
-            <Link to={`/${pathItem}?namespaces=${namespace}`} onClick={this.updateTypeFilter}>
-              {itemName} Type: {istioType}
+      <div className="breadcrumb">
+        <Breadcrumb>
```
Comment (aljesusg) [inline src/components/BreadcrumbView/BreadcrumbView.tsx]: Is there any style missing for this div ?

Comment (xeviknal) [reply src/components/BreadcrumbView/BreadcrumbView.tsx]: This is not part of the PR anymore. Breadcrumb was moved in a previous PR. Outdated code, sorry.

**Comment by abonas** [comment]
the description probably has a minor mistake, it says it's migration to PF3 :)

**Comment by xeviknal** [comment]
@aljesusg @lucasponce @hhovsepy @skondkar this PR is ready to review

File: src/components/ErrorBoundary/ErrorBoundaryWithMessage.tsx [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+interface MessageProps {
+  message: string;
+}
+
+export default class ErrorBoundaryWithMessage extends React.Component<MessageProps> {
+  alert() {
+    return (
+      <div className="card-pf-body">
+        <Alert variant="warning" title={this.props.message || 'Something went wrong rending this component'}>
```
Comment (aljesusg) [inline src/components/ErrorBoundary/ErrorBoundaryWithMessage.tsx]: This style wont' be in PF4, maybe you can change it or add the CardBody component of PF4

Comment (xeviknal) [reply src/components/ErrorBoundary/ErrorBoundaryWithMessage.tsx]: I changed that for bit. I used the Card and CardBody components :)

**Comment by hhovsepy** [comment]
Verified on FF and Chrome.

**Pull request merged**